pvsystemprofiler/algorithms/latitude/estimation.py

```diff
@@ -0,0 +1,8 @@
+import numpy as np
+from pvsystemprofiler.algorithms.latitude.calculation import calc_lat
+
+
+def estimate_latitude(hours_daylight, delta):
```

Comment:
there is only one way of estimating latitude, so this should be a pass-through function